hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileUpdateHandler.java

```diff
@@ -77,12 +77,13 @@ public void handle(@NonNull final HandleContext handleContext) throws HandleExce
 
         final var fileStore = handleContext.writableStore(WritableFileStoreImpl.class);
         final var fileUpdate = handleContext.body().fileUpdateOrThrow();
-        final var maybeFile = requireNonNull(fileStore)
-                .get(fileUpdate.fileIDOrElse(FileID.DEFAULT).fileNum());
+        final var maybeFile = requireNonNull(fileStore).get(fileUpdate.fileIDOrElse(FileID.DEFAULT));
```

Comment:
No reason to have `requireNonNull(fileStore)`, `HandleContext.writableStore` does not return null (it's annotated `@NonNull`), so it can just be `fileStore`.

Reply:
That's not something I changed. I will remove the requireNonNull if I end up checking a different update for something else, but I don't think this is worth holding up the PR for.

Reply:
Agreed, it's a minor item.

Reply:
Fixed